### PR TITLE
Package range.0.3

### PR DESCRIPTION
**Pull request by aldrikfr**
### `range.0.3`
Fold on integer range
Enable folding on integer range with split capacities
for making distributed computing easier with libraries
like functory

---
* Homepage: https://github.com/aldrikfr/range
* Source repo: git+https://github.com/aldrikfr/range
* Bug tracker: https://github.com/aldrikfr/range/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/range/range.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+like functory
+"""
+maintainer : "Aldrik KLEBER <contact@aldrik.net>"
+authors : "Aldrik KLEBER"
+license : "gpl-v3"
+homepage : "https://github.com/aldrikfr/range"
+dev-repo: "git+https://github.com/aldrikfr/range"
+bug-reports :"https://github.com/aldrikfr/range/issues"
+depends: [
+"ocaml" {>= "4.07.0"}
```
Comment (mseri) [inline packages/range/range.0.3/opam]: Why such a strict bound on ocaml? I did not seem to find any code that requires it

File: packages/range/range.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+"ocamlfind"
```
Comment (mseri) [inline packages/range/range.0.3/opam]: I think this is not needed

Comment (aldrikfr) [reply packages/range/range.0.3/opam]: agree I remove it

**Comment by mseri** [comment]
Thanks! Looks good

**Comment by kit-ty-kate** [comment]
Sorry I missed the previous PR. See the last comment there. This packages doesn't do anything. Please fix it.

**Comment by kit-ty-kate** [comment]
see https://github.com/ocaml/opam-repository/pull/13033

**Comment by mseri** [comment]
Good point. Sorry I had missed that

**Pull request merged**